docs/3_advanced.rst

```diff
@@ -24,6 +24,25 @@ the platform-specific Query classes can be used.
 You can use these query classes as a drop in replacement for the default ``Query`` class shown in the other examples.
 Again, if you encounter any issues specific to a platform, please create a GitHub issue on this repository.
 
+Or even different query languages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some services created their own query language similar to SQL. To generate expressions for Jira there is a ``JiraQueryBuilder`` class.
+
+.. code-block:: python
+
+    from pypika import JiraTable, JiraQueryBuilder
```

Comment:
Should the example use `JiraQuery` instead of `JiraQueryBuilder`?

Reply:
Think that that would provide consistency with the rest of the API

Reply:
To be honest I just don't know how to make in consistent.
There are no such thing as tables (so JiraTable is just a dummy placeholder).

Best thing I can do is probably create a bunch of overrides for `from_`, `into` and other methods which would return `cls._builder(**kwargs)`

Is it ok?
```python
>>> table = JiraTable()
>>> JiraQuery._builder().where(table.project.isin(["PROJ1", "PROJ2"]))
project IN ("PROJ1","PROJ2")
```

Reply:
Hm. I guess `JiraQuery.from_(table).where(table.project.isin(["PROJ1", "PROJ2"]))` looks kind consistent.

Reply:
done

Reply:
Maybe the `JiraQuery` could have a `where` method as well that initializes table. 

Personally, I tend to used `Field`s 

Maybe it would read like this

```python
query = (
    JiraQuery
    .where(
        (Field("project").isin(["PROJ1", "PROJ2"]))
        & (Field("issuetype") == "my_issue_type")
    )
)

Reply:
Done.
Working in both ways:
```python
>>> JiraQuery.where(Field("project").isin(["PROJ1", "PROJ2"]) & (Field("issuetype") == "my_issue_type"))
project IN ("PROJ1","PROJ2") AND issuetype="my_issue_type"
>>> table = JiraTable()
>>> JiraQuery.where(table.project.isin(["PROJ1", "PROJ2"]) | table.project == 2)
project IN ("PROJ1","PROJ2") OR project=2
```